static/app/views/settings/project/projectReleaseTracking.spec.tsx

```diff
@@ -43,17 +44,18 @@ describe('ProjectReleaseTracking', function () {
     jest.clearAllMocks();
   });
 
-  it('renders with token', function () {
+  it('renders with token', async function () {
     render(
       <ProjectReleaseTracking
         organization={org}
         project={project}
         plugins={{loading: false, plugins: PluginsFixture()}}
-        {...routerProps}
       />
     );
 
-    expect(screen.getByRole('textbox')).toHaveValue('token token token');
+    await waitFor(() => {
+      expect(screen.getByRole('textbox')).toHaveValue('token token token');
+    });
```

Comment:
this is not a findBy because the input has a different placeholder while loading